pychunkedgraph/app/segmentation/v1/routes.py

```diff
@@ -163,6 +163,15 @@ def handle_children(table_id, node_id):
     return jsonify_with_kwargs(resp, int64_as_str=int64_as_str)
 
 
+### RANGE READ ------------------------------------------------------------------
+
+@bp.route("/table/<table_id>/range_read/<chunk_id>", methods=["GET"])
+@auth_requires_permission("view")
+def handle_range_read(table_id, chunk_id):
+    range_read = common.handle_range_read(table_id, chunk_id)
+    return pickle.dumps(range_read)
```

Comment:
what prevents us from formatting this as a json endpoint?
I am worried that this here is getting too much for higher layers. Could we make one that is limited to layer 2 for now?

Reply:
Sounds good on limiting it to layer 2.

Nothing prevents it from being a JSON endpoint, I just figured since it can be a decent amount of data that we'd want binary, and looked at the get_roots endpoint for inspiration. Maybe there should be 1 JSON and 1 binary?

Reply:
Agreed. There should be a specific binary and a json one. Would use the `tobinary` function we have for that though